data.sql

```diff
@@ -244,3 +243,26 @@ VALUES ((SELECT id FROM animals WHERE name = 'Blossom'), (SELECT id FROM vets WH
 
 INSERT INTO visits (animal_id, vet_id, visit_date)
 VALUES ((SELECT id FROM animals WHERE name = 'Blossom'), (SELECT id FROM vets WHERE name = 'William Tatcher'), '2021-01-11');
+*/
+
+-- Add an email column to your owners table
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+-- This will add 3.594.280 visits considering you have 10 animals, 4 vets, and it will use around ~87.000 timestamps (~4min approx.)
+INSERT INTO visits (animal_id, vet_id, visit_date) SELECT * FROM (SELECT id FROM animals) animal_ids, (SELECT id FROM vets) vets_ids, generate_series('1970-01-01'::timestamp, '2021-01-01', '4 hours') visit_timestamp ON CONFLICT DO NOTHING;
+
+-- This will add 2.500.000 owners with full_name = 'Owner <X>' and email = 'owner_<X>@email.com' (~2min approx.)
+insert into owners (full_name, email) select 'Owner ' || generate_series(1,2500000), 'owner_' || generate_series(1,2500000) || '@mail.com';
+
+-- First improve on performance
+CREATE INDEX ON visits (animal_id);
+
+-- Second improve
+CREATE INDEX ON visits (vet_id);
+CREATE TEMPORARY TABLE temp_visits AS SELECT * FROM visits WHERE vet_id = 2;
+CREATE INDEX ON temp_visits (vet_id);
+EXPLAIN ANALYZE SELECT * FROM temp_visits;
+
+-- Third improve
+CREATE INDEX ON owners (email);
+
```

Comment:
- [ ] @redmarverick and @Mar12358 Great work improving the queries speed by adding  indexes, however, the code to add indexes should be in the schema file, since it makes changes to the database schema